EndlessSky.xcodeproj/project.pbxproj

```diff
@@ -27,7 +27,6 @@
 		62A405BA1D47DA4D0054F6A0 /* FogShader.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 62A405B81D47DA4D0054F6A0 /* FogShader.cpp */; };
 		62C3111A1CE172D000409D91 /* Flotsam.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 62C311181CE172D000409D91 /* Flotsam.cpp */; };
 		6A5716331E25BE6F00585EB2 /* CollisionSet.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 6A5716311E25BE6F00585EB2 /* CollisionSet.cpp */; };
-		938E07E4235970CE00DC2C2B /* FormationPattern.cpp in Sources */ = {isa = PBXBuildFile; fileRef = 938E07E0235970CE00DC2C2B /* FormationPattern.cpp */; };
```

Comment:
Was this file double, or do we really not like it? (And if it were double, shouldn't we then just fix that in a separate PR?)

Reply:
Looks like it is doubled, as it also exists on line 37.
I imagine this was deleted because the xCode patch file that got generated caught it, and Hurl simply applied applied the patch file. I do the same whenever I need to update this file, and just assume whatever the patch file says is how it should be, even if there appear to be unrelated changes.

Reply:
Ok, cool. Then I'm fine with getting it removed here.